keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/plugin/TaskLauncher.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.keel.plugin
+
+import com.netflix.spinnaker.keel.api.Resource
+import com.netflix.spinnaker.keel.api.application
+import com.netflix.spinnaker.keel.api.id
+import com.netflix.spinnaker.keel.api.serviceAccount
+import com.netflix.spinnaker.keel.events.Task
+import com.netflix.spinnaker.keel.model.EchoNotification
+import com.netflix.spinnaker.keel.model.Job
+import com.netflix.spinnaker.keel.model.OrchestrationRequest
+import com.netflix.spinnaker.keel.model.OrchestrationTrigger
+import com.netflix.spinnaker.keel.model.toEchoNotification
+import com.netflix.spinnaker.keel.orca.OrcaService
+import com.netflix.spinnaker.keel.persistence.DeliveryConfigRepository
+import org.slf4j.LoggerFactory
+
+/**
+ * Wraps [OrcaService] to make it easier to launch tasks in a standard way.
+ */
+class TaskLauncher(
+  private val orcaService: OrcaService,
+  private val deliveryConfigRepository: DeliveryConfigRepository
+) {
+  suspend fun submitJobToOrca(
+    resource: Resource<*>,
+    description: String,
+    correlationId: String,
+    job: Map<String, Any?>
```

Comment:
Should we make this a `vararg` and rename the function to `submitJobsToOrca` in case we ever want to pass in more than one job?

Reply:
Yeah. I was thinking the same, but I figure we can do it any time.